core/src/main/kotlin/org/evomaster/core/parser/GenePostgresLikeVisitor.kt

```diff
@@ -26,7 +26,10 @@ class GenePostgresLikeVisitor : PostgresLikeBaseVisitor<VisitResult>() {
         }
 
         val disjunction = DisjunctionRxGene("disj", terms, true, true)
-        val gene = RegexGene("LIKE", DisjunctionListRxGene(listOf(disjunction)))
+
+        val text = RegexUtils.getRegexExpByParserRuleContext(ctx)
+
+        val gene = RegexGene("LIKE$text", DisjunctionListRxGene(listOf(disjunction)))
```

Comment:
for readability, i guess should put a space? ie, `LIKE $text`. this applies to all other cases here